test/caiquenator_web/controllers/page_controller_test.exs

```diff
@@ -0,0 +1,12 @@
+defmodule CaiquenatorWeb.PageControllerTest do
+  use CaiquenatorWeb.ConnCase
+
+  test "GET /", %{conn: conn} do
+    conn = get(conn, "/")
+    html = html_response(conn, 200)
+
+    phrase = Floki.find(html, "p.lead") |> Floki.text() |> String.trim()
```

Comment:
Now that we have more then one `|>` operator we can use it in the more elixir way, wdyt?

```elixir
phrase = 
  html 
  |> Floki.find("p.lead")
  |> Floki.text() 
  |> String.trim()
```